src/vcFBX.cpp

```diff
@@ -365,7 +369,7 @@ vdkError vcFBX_Open(vdkConvertCustomItem *pConvertInput, uint32_t everyNth, cons
 
   if (result != vE_Success)
   {
-    pFBX->pManager->Destroy(); // Destroying manager destroys all objects that were created with it
+    pFBX->pManager->Destroy();
```

Comment:
Is this comment no longer true?

Reply:
Never was quite true I believe that comment was leftover from johnathon